test/src/test/java/jenkins/security/ClassFilterImplTest.java

```diff
@@ -161,6 +161,32 @@ public void moduleClassesShouldBeWhitelisted() throws Exception {
         filter.check("org.jenkinsci.main.modules.instance_identity.PageDecoratorImpl");
     }
 
+    @Test
+    @Issue("JENKINS-51326")
+    public void primitivesShouldBeWhitelisted() throws Exception {
+        ClassFilterImpl filter = new ClassFilterImpl();
+
+        // ClassFilter#isBlacklisted(String) is noop now, but we check it just in case it changes
```

Comment:
I don't understand this comment. It seems to imply a check for string, but I don't see one.

Reply:
I just meant that the invocations below do nothing and hence they pass independently of what you pass

Reply:
Ah, okay. i didn't read it closely enough.